wasm/sdk_test.go

```diff
@@ -0,0 +1,174 @@
+package wasm
+
+import (
+	// "encoding/json"
+	"fmt"
+	"strings"
+	"syscall/js"
+	"testing"
+
+	"github.com/0chain/gosdk/zboxcore/blockchain"
+)
+
+var storageConfig = fmt.Sprintf("{\"wallet\":%s,\"signature_scheme\":\"bls0chain\"}", walletConfig)
+
+var chainConfig = "{\"chain_id\":\"0afc093ffb509f059c55478bc1a60351cef7b4e9c008a53a6cc8241ca8617dfe\",\"block_worker\":\"http://127.0.0.1:1/dns\",\"miners\":[\"http://127.0.0.1:1/miner01\"],\"sharders\":[\"http://127.0.0.1:1/sharder01\"],\"signature_scheme\":\"bls0chain\",\"min_submit\":50,\"min_confirmation\":50,\"confirmation_chain_length\":3,\"eth_node\":\"https://ropsten.infura.io/v3/f0a254d8d18b4749bd8540da63b3292b\",\"num_keys\":\"1\"}"
+
+func TestInitializeConfig(t *testing.T) {
+	setup(t)
```

Comment:
For testing please use sub-tests, like t.Run("test name", t {}) ... it will avoid many duplications and confusions